.env

```diff
@@ -0,0 +1,5 @@
+DB_USERNAME='nostalgia'
+DB_PASSWORD='N0$ta1gia!'
+DB_DATABASE='nostalgia'
+DB_HOST='localhost'
+DB_DIALECT='mysql'
```

Comment:
That's a big nope

Reply:
Damnit. Thought i killed that on last round. I missed this on the initial .gitignore and added afterward. But it keeps coming back... How do i keep .env out of here?

Reply:
Once it has been committed it can't be ignored. You have to delete it. Commit that deletion. Then you can add it to .gitignore and add the file back safely